plugins/language_typst.lua

```diff
@@ -0,0 +1,196 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+local style = require "core.style"
+
+-- This syntax is probably very borken, so feel free to improve it.
+
+--region add other styles
+local new_styles = {"bold", "italic", "bold_italic", "underline"}
+
+local code_font_size = style.code_font:get_size()
+local code_font_path = style.code_font:get_path()
+
+if type(code_font_path) == "table" then
+    code_font_path = code_font_path[1]
+end
+
+for _, attr in pairs(new_styles) do
+  local attributes = {}
+  if attr ~= "bold_italic" then
+    attributes[attr] = true
+  else
+    attributes["bold"] = true
+    attributes["italic"] = true
+  end
+  -- yes way to copy user custom font with additional attributes :)
+  style.syntax_fonts["typst_"..attr] = renderer.font.load(
+    code_font_path,
+    code_font_size,
+    attributes
+  )
+end
```

Comment:
I'm sorry for this, but could you avoid adding other token types?
The code as it is now will probably work, but will be broken for people that customize their fonts (even at runtime).

We intend to add better support for this use-case in the future, but it's not there yet.

Reply:
Sure, but is there another way I can make the text bold, or italic (or even underlined, even though I could not get that to work in the first place) ?

Reply:
Sadly not yet. But it is planned for the future.